lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -3095,80 +2703,64 @@ public function supportsColumnLengthIndexes() : bool
 
     /**
      * Whether the platform supports altering tables.
-     *
-     * @return bool
      */
-    public function supportsAlterTable()
+    public function supportsAlterTable() : bool
     {
         return true;
     }
 
     /**
      * Whether the platform supports transactions.
-     *
-     * @return bool
      */
-    public function supportsTransactions()
+    public function supportsTransactions() : bool
     {
         return true;
     }
 
     /**
      * Whether the platform supports savepoints.
-     *
-     * @return bool
      */
-    public function supportsSavepoints()
+    public function supportsSavepoints() : bool
     {
         return true;
     }
 
     /**
      * Whether the platform supports releasing savepoints.
-     *
-     * @return bool
      */
-    public function supportsReleaseSavepoints()
+    public function supportsReleaseSavepoints() : bool
     {
         return $this->supportsSavepoints();
     }
 
     /**
      * Whether the platform supports primary key constraints.
-     *
-     * @return bool
      */
-    public function supportsPrimaryConstraints()
+    public function supportsPrimaryConstraints() : bool
     {
         return true;
     }
 
     /**
      * Whether the platform supports foreign key constraints.
-     *
-     * @return bool
      */
-    public function supportsForeignKeyConstraints()
+    public function supportsForeignKeyConstraints() : bool
     {
         return true;
     }
 
     /**
      * Whether this platform supports onUpdate in foreign key constraints.
-     *
-     * @return bool
      */
-    public function supportsForeignKeyOnUpdate()
+    public function supportsForeignKeyOnUpdate() : bool
     {
-        return $this->supportsForeignKeyConstraints() && true;
```

Comment:
What was this for? A boolean cast?

Reply:
This is the only purpose I can think of. As long as the return type is now enforced, it's no longer needed.